code/modules/mob/living/simple_animal/hostile/terror_spiders/black.dm

```diff
@@ -0,0 +1,38 @@
+
+#define SPINNING_WEB 1
```

Comment:
you shouldn't redefine things every file, that defeats the purpose of defines
make the file
`code/modules/mob/living/simple_animal/hostile/terror_spiders/__defines.dm`
and put these there
if you insist on having `#undef` statements, put it in `terror_spiders/undef.dm`